spec/models/application_record_spec.rb

```diff
@@ -104,6 +104,21 @@
     end
   end
 
+  describe 'interview_count' do
+    before :each do
+      record_1 = create :application_record
+      record_2 = create :application_record
+      create :interview, application_record: record_1
+      @collection = ApplicationRecord.where(id: record_1.id)
```

Comment:
An ActiveRecord scope is a method defined on ActiveRecord relations of a particular class. The code is correct.

http://guides.rubyonrails.org/active_record_querying.html#scopes